client/state/selectors/get-site-wordads-status.ts

```diff
@@ -0,0 +1,15 @@
+import { get } from 'lodash';
+import type { AppState } from 'calypso/types';
+
+/**
+ * Returns the wordads status for a site
+ */
+export default function getSiteWordadsStatus(
+	state: AppState,
+	siteId: number | undefined | null
+): string | null {
+	if ( ! siteId ) {
+		return null;
+	}
+	return get( state, [ 'wordads', 'status', siteId, 'status' ], null );
```

Comment:
Calypso folks have been hard at work for several years now to remove lodash from the codebase. Could you use regular JS here?

```suggestion
	return state.wordads?.status?.[ siteId ]?.status ?? null;
```